utils/python/CIME/create_test.py

```diff
@@ -228,10 +251,12 @@ def _update_test_status(self, test, phase, status):
                    "New phase should be set to pending status")
             expect(self._phases.index(old_phase) == phase_idx - 1,
                    "Skipped phase?")
-
+        if(self._cime_model == "cesm"):
```

Comment:
Does this really need to be updated every time a state changes or can it just be updated at the end?